ha-example/README.md

```diff
@@ -0,0 +1,160 @@
+# Flink cluster HA configuration
+
+In the deployment of Flink cluster, it consists of operators, job managers, and task managers.
+By default, the deployment is not a high availability (HA) cluster, we need to configure it to achieve it.
+
+## Flink operator
+
+The Flink operator manages and operates Flink Deployments. It supports high availability through leader election and standby operator instances.
+To enable leader election we need to add the following two mandatory operator configuration parameters.
+
+```
+kubernetes.operator.leader-election.enabled: true
+kubernetes.operator.leader-election.lease-name: flink-operator-lease
+```
+The lease name must be unique in the current lease namespace.
+
+So, when installing flink operator using helm, we need to add these 2 configs + existing default configs via command like this:
```

Comment:
```suggestion
When installing the Flink Kubernetes operator using `helm`, we need to add these 2 parameters into the existing default configuration string.
This can be done via the command line using the `--set` flag:
```
I realise this is a consequence of how the config passing is set up in the operator, but this is not great UX. If these other options are truely _default_ then they should be in the `helm/flink-kubernetes-operator/conf/flink-conf.yaml`. That way only user supplied defaults could be set via the helm cli/custom values file.

Might be a good improvement to the helm chart upstream.

Reply:
+1! Will work on that!